advanced/dapps/react-dapp-v2/src/modals/RequestLoaderModal.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
```

Comment:
looks very similar to the old one - why a new one?

Reply:
its just some text changes on the modal, it don't need any approval or reject action on wallet side.
I can refactor the old one to take these messages as props.